tests/test_all_fields.py

```diff
@@ -145,9 +225,7 @@
     'invoices': {'updated'},
     'plans': {'updated'},
     'invoice_line_items': {
-        'updated',
```

Comment:
the `updated` and `subscription_item` field is added by tap. So, removed it from the `KNOWN_MISSING_FIELDS` map.

Reply:
No, the sync strategy is not changed. But, I verified that the above fields are not available in the records by running sync mode for the `invoice_line_items` stream. Sync for the child stream works based on parent records. That's why the above fields are removed from the `FIELDS_ADDED_BY_TAP` map.